test/framework/src/main/java/org/elasticsearch/cluster/coordination/DeterministicTaskQueue.java

```diff
@@ -405,7 +405,18 @@ public Object get(long timeout, TimeUnit unit) {
 
             @Override
             public Cancellable scheduleWithFixedDelay(Runnable command, TimeValue interval, String executor) {
-                throw new UnsupportedOperationException();
+                // TODO: Implement fully like schedule
+                return new Cancellable() {
```

Comment:
Added dummy return only here for now. This is used to schedule a task `org.elasticsearch.indices.IndexingMemoryController.ShardsIndicesStatusChecker` in `org.elasticsearch.indices.IndexingMemoryController`. Just a dummy for now sine that task doesn't seem relevant for this test.

Reply:
Done in https://github.com/elastic/elasticsearch/pull/36644/commits/2fa91b32d80c96c6a0e1520c289ef5c56c0aeb6b.